graphix/pattern.py

```diff
@@ -1234,9 +1234,11 @@ def draw_graph(
         self,
         node_distance=(1, 1),
         figsize=None,
+        gflow_from_pattern=False,
```

Comment:
perhaps `gflow_from_pattern` should be defaulted to True

Reply:
I set the default to True.